src/js/frontend/src/Redux/reducer.js

```diff
@@ -20,16 +24,37 @@ export function rootReducer (state = IMap(), action) {
 
 const defaultState = {
   showAboutScreen: false,
+  entities: {
+    people: {},
+    organisations: {},
+    "government-offices": {},
+  },
 };
 
-export function aboutReducer(state = defaultState, {type}) {
-  let newState = Object.assign({}, state);
-
+function aboutReducer(state = defaultState.showAboutScreen, {type}) {
   switch (type) {
     case actionTypes.TOGGLE_ABOUT:
-      newState.showAboutScreen = !newState.showAboutScreen;
-      return newState;
+      return ! state;
```

Comment:
oooh battle of spaces, yeah I'll change the eslint when I get a chance 
welcome to your new life